fastwarc/fastwarc/__init__.py

```diff
@@ -15,3 +15,14 @@
 from .stream_io import FileStream, GZipStream, LZ4Stream
 from .stream_io import FastWARCError, StreamError
 from .warc import ArchiveIterator, WarcRecord, WarcRecordType
+
+__all__ = [
+    "FileStream",
+    "GZipStream",
+    "LZ4Stream",
+    "FastWARCError",
+    "StreamError",
+    "ArchiveIterator",
+    "WarcRecord",
+    "WarcRecordType"
+]
```

Comment:
Since there aren't any unwanted global imports in this file, I didn't specify `__all_` explicitly. Is this needed for mypy? Otherwise I would suggest to remove it.

Reply:
Hmm, ok. It should be said that these are there only for legacy compatibility and explicit submodule imports should be preferred.